src/test.mjs

```diff
@@ -851,8 +822,8 @@ t.test('Exceed max file size.', async t => {
       })
     )
 
-    body.append(1, fs.createReadStream(TEST_FILE_PATH_JPG))
-    body.append(2, fs.createReadStream(TEST_FILE_PATH_JSON))
```

Comment:
The test tests:
1. That an upload that exceeds the max file size triggers an error
2. That a subsequent upload does not error

Because both uploads now exceed the max file size, they both error, which is the correct behavior. We could either introduce another smaller test file, or remove the second half of the test.

Reply:
I'll add a second smaller test file.